promise.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var Promise = require('bluebird');
```

Comment:
As `Promises` are now available on native node, it'd make more sense if  `bluebird` is conditionally required?